index.html

```diff
@@ -18,16 +21,178 @@
             </div>
         </nav>
     </header>
-    <!-- END NAVBAR SECTION-->
+    <!-- End NAVBAR SECTION-->
 
-    <!-- START HEADLINE SECTION-->
+    <!-- Start HEADLINE SECTION-->
     <section class="headline">
         <div class="content">
             <h1 class="author">Thiri Htet Htet Aung</h1>
             <p class="intro">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
             <button type="button" class="collab-btn">Start collaboration</button>
         </div>
     </section>
-    <!-- END HEADLINE SECTION -->
+    <!-- End HEADLINE SECTION -->
+
+    <!-- Start WORK SECTION -->
+    <section class="works">
+        <!-- start title -->
+        <div class="title">
+            <h1>Projects</h1>
+        </div>
+        <!-- end title -->
+
+        <!-- start project card -->
+        <div class="projects">
+            <!-- start card1 -->
+            <div class="project-cards project1">
+                <div class="pjimgs">
+                    <img src="./img/pj1.svg" alt="pj1"/>
+                </div>
+                <div class="pjcontent">
+                    <p class="pjname">Project name goes here</p>
```

Comment:
Having a proper ranking of contents in mind, it is advisable that all occurrences of the opening heading for your cards should be a heading tag (at least an `h3` tag as opposed to using a `p` tag).